src/app/__init__.py

```diff
@@ -1,26 +1,27 @@
-import os
 
-from dotenv import load_dotenv
+from app import routes
 from flask import Flask, request
+from dotenv import load_dotenv
+
+from .models.timelinepost import TimelinePost
+import os
```

Comment:
Built-in python libraries goes at the top

Reply:
Thanks for letting me know 👍